charts/flink-tools/templates/FlinkCluster.yaml

```diff
@@ -22,6 +22,12 @@ spec:
         requests:
           storage: {{ .Values.localStorage.size | quote }}
       storageClassName: standard
+{{- if .Values.enableExtraVolume }}
```

Comment:
This should be changed to support multiple NFS volumes. Change this to:
```
  volumes:
{{ toYaml .Values.volumes | indent 4 }}
```

Then in /charts/flink-tools/values.yaml, specify an empty default:
```
volumes: {}
```

Reply:
changed accordingly